index.rst

```diff
@@ -42,17 +42,23 @@ Avro schemas can be composed of nested sub-schemas under a top level namespace.
 Nesting simplifies what would otherwise be monolithic schemas as new fields are added.
 For example, the base alert schema (``lsst.alert``) is of type "record" and includes previous detections of DIA sources as an array of type ``lsst.alert.diaSource``.
 
-The current schema proposed for use with LSST alerts is stored in the `lsst/alert_packet`_ repository.
-This contains all fields specified by the LSST Data Products Definition Document (LSE-163; :cite:`LSE-163`).
+The current schemas contain all fields specified by the LSST Data Products Definition Document (:cite:`LSE-163`).
 At this stage in construction, this schema should be regarded as exploratory and subject to rapid change; as we move closer to the operational era, a change control process will be implemented.
 
+The current schema proposed for use with LSST alerts is stored in the `lsst/alert_packet`_ repository.
+This primary purpose of this package is to enable generation of alerts during Prompt Processing.
+Versioning of this package with the rest of the Science Pipelines software ensures reproducability.
+During operations, users will access the production schemas using other tools (see :ref:`alertManagement`).
+
 .. _lsst/alert_packet: https://github.com/lsst/alert_packet
 
+.. _alertManagement:
+
 Management and Evolution
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 Avro packets may be shipped with or without an embedded schema.
-Since the schema can be lengthy, and since each alert will be packaged for shipping separately, we expect to distribute alerts separately from the associated schema.
+Since the schema can be lengthy, and since each alert will be packaged for shipping separately, we expect to distribute alerts to community alert brokers without including the associated schema.
 
 The recipient of each packet requires a schema to interpret the data they have been sent.
 In the simplest model, the producer (the LSST Alert Distribution System) and the consumer (the remote broker or science user) simply agree on the schema prior to the start of the stream.
```

Comment:
Perhaps remove `or science user` from this line, or revise all the way to `and the remote broker simply agree`.